rds/variables.tf

```diff
@@ -11,12 +11,22 @@ variable "ca_cert_identifier" {
   type    = string
 }
 
+variable "backup_retention_period" {
+  default = 7
+  type    = number
+}
+
 variable "instance_class" { default = "db.t3.micro" }
 variable "engine_version" {
   default = "14"
   type    = string
 }
-variable "allocated_storage" { default = 100 }
+variable "allocated_storage" { default = 10 }
+
+variable "max_allocated_storage" {
```

Comment:
For clarity: can we name this e.g. `storage_autoscaling_uppler_limit`? "max_allocated_storage" does not really tell me what it stands for (I think this is bad naming from Terraform).